cli/src/main.rs

```diff
@@ -116,7 +117,7 @@ struct Cli {
     command: Commands,
 }
 
-#[tokio::main]
+#[tokio::main(flavor = "current_thread")]
```

Comment:
Is there a specific reason for this change?

Reply:
Because this simple CLI app doesn't need a multi threaded runtime to do basic zbus calls